proposals/wgs/wg-lifecycle.md

```diff
@@ -0,0 +1,92 @@
+## SUMMARY:
+
+This document covers everything you need to know about the creation and retirement (“lifecycle”) of a working group within Kubeflow. General project governance information can be found in the [governance doc].
+Out of scope for this document: [subproject] creation.
+
+[Creation]
+[Retirement]
+
+## [Creation]
+### Prerequisites for a WG
+- [ ] Read [wg-governance.md]
+- [ ] Ensure all WG Chairs, Technical Leads, and other leadership roles are [community members]
+- [ ] Send an email to the Steering Committee <steering@kubeflow.org> to scope the WG and get provisional approval.
+- [ ] Look at the checklist below for processes and tips that you will need to do while this is going on. It's best to collect this information upfront so you have a smoother process to launch
+- [ ] Follow the [WG charter process] to propose and obtain approval for a charter
+- [ ] Announce new WG on kubeflow-discuss@googlegroups.com
+
+### [GitHub]
+
+TODO: This section needs to be added for KF. In particular do we want to use [generator doc] which is what Kubernetes uses or is that
+too heavy weight?
+
+- [ ] Submit a PR that will add rows to [wgs.yaml] using the [generator doc]; this will create README files and OWNERS_ALIASES files for your new directory in `kubernetes/community`
+- You’ll need:
+  - WG Name
+  - Directory URL
+  - Mission Statement
+  - Chair Information
+  - Meeting Information
+  - Contact Methods
+  - Any WG Stakeholders
+  - Any Subproject Stakeholders
+- [ ] Add WG-related docs like charter.md, schedules, roadmaps, etc. to your new kubeflow/community/wg-foo directory once the above PR is merged.
+- [ ] File a [kubeflow/testing] Issue  and PR for a label; read about our [GitHub management] services
+
+### Communicate:
+
+WGs are responsible for setting up and administering the following communication channels
+- [ ] Mailing lists 
+      * foo@googlegroups.com
```

Comment:
Since slack is supposed to be wg-foo, should ML follow the same pattern?

Reply:
I don't have a strong oppinion. What do others think?